### ISSUE TITLE
Tapping the map sometimes taps an invisible map callout from a previous cache

### ISSUE THREAD

**Issue by cachly** [cachly/bug-tracker#1575]
**[Original report](https://bitbucket.org/zedsaidstudio/cachly/issues/1575) by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

**Comment by cachly** [comment]
**Original comment by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

I have had the same but only over the last week. Need to enlarge map considerably to move prior cache out of view and then tapping on another cache will work. It happens as you pick a cache from the map, then log it and once you have come out of the cache and are back on the map the previous cache comes up time and time again even though my finger is nowhere near the previous cache

**Comment by cachly** [comment]
**Original comment by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

I just experienced the problem again. Found a cache and logged it. Clicked on a new cache on the map and previous cache description comes up. I tried a few times, eventually the prior cache "released". It seemed to be at the same time I got far enough away from the cache that it was outside the pulsing green circle around my location. That may explain why I experienced the problem repeatedly yesterday but could not reproduce last night.
